pkg/common/utils/resource/pod.go

```diff
@@ -678,6 +688,50 @@ func getMultiConfigVolumeAndVolumeMount(cmInfo *v1.ConfigMapInfo, componentType
 	return volumes, volumeMounts
 }
 
+func getMultiSecretVolumeAndVolumeMount(bSpec *v1.BaseSpec, componentType v1.ComponentType) ([]corev1.Volume, []corev1.VolumeMount) {
+	var volumes []corev1.Volume
+	var volumeMounts []corev1.VolumeMount
+
+	if bSpec.Secrets == nil {
```

Comment:
this check is not needed. for golang splice, iterate the nil splice is ok.

Reply:
Done